pkg/operator/ceph/disruption/nodedrain/reconcile.go

```diff
@@ -121,20 +121,20 @@ func (r *ReconcileNode) reconcile(request reconcile.Request) (reconcile.Result,
 			var deployment appsv1.Deployment
 			err := r.client.List(context.TODO(), deploymentList, client.MatchingLabels(labelSelector), client.InNamespace(osdPod.GetNamespace()))
 			if err != nil || len(deploymentList.Items) < 1 {
-				logger.Errorf("Cannot find deployment for osd id %s in namespace %s", labels[osd.OsdIdLabelKey], osdPod.GetNamespace())
+				logger.Errorf("cannot find deployment for osd id %q in namespace %q", labels[osd.OsdIdLabelKey], osdPod.GetNamespace())
 			} else {
 				deployment = deploymentList.Items[0]
 			}
 			if len(deploymentList.Items) > 1 {
-				logger.Errorf("Found multiple deployments for osd id %s in namespace %s: %+v", labels[osd.OsdIdLabelKey], osdPod.GetNamespace(), deploymentList)
+				logger.Errorf("found multiple deployments for osd id %q in namespace %q. %+v", labels[osd.OsdIdLabelKey], osdPod.GetNamespace(), deploymentList)
```

Comment:
I think this one might actually make more sense to be a colon, not a period since it is a struct, not an err.
```suggestion
				logger.Errorf("found multiple deployments for osd id %q in namespace %q: %+v", labels[osd.OsdIdLabelKey], osdPod.GetNamespace(), deploymentList)
```